File: backend/src/main/java/catulvir/beerapp/backend/model/Beer.java
package catulvir.beerapp.backend.model;

import lombok.*;
import java.util.List;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class Beer {

    private Long id;

	private List<Flavour> flavours;

	private BeerType beerType;

	private OriginCountry originCountry;

	private Manufacturer manufacturer;

	@Size(max = 63)
	private String name;

	private String routePath;

	@Size(max = 511)
	private String description;
	
	@Size(max = 511)
	private String imageLink;

	@NotNull
	private Boolean isCraft;

	@NotNull
	private Float abv;

	private Float averageRating;

	private Float bitterness;

	private Float colour;

	private Float wortDensity;

	private Integer lowerServeTemperature;

	private Integer higherServeTemperature;

	//private String nutritionFacts;
}

File: frontend/angular/beer-app/src/app/beers/beer.ts
import { BeerType } from "../beer-types/beer-type";
import { Flavour } from "../flavours/flavour";
import { Manufacturer } from "../manufacturers/manufacturer";
import { OriginCountry } from "../origin-countries/origin-country";

export class Beer {
    
    id?: number;
    beerType?: BeerType;
    originCountry?: OriginCountry;
    manufacturer?: Manufacturer;
    flavours?: Flavour[];
    name?: string;
    description?: string;
    imageLink?: string;
	isCraft?: boolean;
	abv?: number;
	averageRating?: number;
	bitterness?: number;
    colour?: number;
	wortDensity?: number;
	lowerServeTemperature?: number;
    higherServeTemperature?: number;

	//private String nutritionFacts;
	//private type picture;
}